src/components/SuperflowsSidebar.tsx

```diff
@@ -363,7 +362,7 @@ export default function SuperflowsSidebar(props: {
                         ) {
                           if (
                             devChatContents[idx - 1].role === "function" ||
-                            devChatContents[idx + 1].role === "function"
+                            devChatContents[idx + 1]?.role === "function"
```

Comment:
is this a fix for the serious crashing we were seeing?

Reply:
Yep